Documentation/common-issues.md

```diff
@@ -266,17 +266,22 @@ What is happening here is that the MON pods are restarting and one or more of th
 
 The `dataDirHostPath` setting specifies a path on the local host for the CEPH daemons to store configuration and data. Setting this to a path like `/var/lib/rook`, reapplying your Cluster CRD and restarting all the CEPH daemons (MON, MGR, OSD, RGW) should solve this problem. After the CEPH daemons have been restarted, it is advisable to restart the [rook-tool pod](./toolbox.md).
 
-# Only a single monitor pod starts
+# Only monitor pods start
```

Comment:
Isn't there a `one` missing? `Only one monitor`?

How about `Mon Pods are (partially) not starting`?